src/database/migrations/20180521093455_create_translationKey_translationValue.js

```diff
@@ -45,6 +45,7 @@ export const up = async knex => {
       .unsigned()
       .references('id')
       .inTable('translationKeys')
+      .onDelete('CASCADE')

```

Comment:
editing existing indexes is really hard :/

Reply:
YOLO